### PR TITLE
add waffle.io statement in contributing.md

### LINKED ISSUE #52: add waffleio note to contributing.md

### PR DESCRIPTION
**Pull request by benbieler**
add waffle.io statement in contributing.md

Resolves #52

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,6 +49,11 @@ If there's a label added to your PR, the commit message should be renamed to the
 The pull request should have some content, too.
 You just need to tell what has changed.
 
+Please add: 
+    ```Resolves #{issue}``` in every pull request.
+    
+This is needed, so that laravel can attach the PR to the issue card.
```
Comment (Ma27) [inline CONTRIBUTING.md]: I guess you've ment waffle.io ;)

**Comment by Ma27** [comment]
could you please add the ticket number (Enhancement #{issue number}) in front of it and squash this change with the changes of the review fixes?

**Comment by benbieler** [comment]
Sure

**Comment by Ma27** [comment]
thank you

**Comment by Ma27** [comment]
squashed locally, thank you :+1:

**Pull request merged**